File: Producto/backend/src/main/java/com/proyecto/macrofit/rutinaejercicio/model/RutinaUsuarioHistorialDTO.java
package com.proyecto.macrofit.rutinaejercicio.model;

import java.time.LocalDate;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class RutinaUsuarioHistorialDTO {
    private Integer idRutinaUsuario;
    private Integer idRutina;
    private String nombreRutina;
    private LocalDate fechaInicio;
    private LocalDate fechaFin;
    private Boolean activo;
}

File: Producto/MacroFit/app/src/main/java/com/duoc/macrofit/rutinas/model/RutinaUsuarioHistorialDTO.kt
package com.duoc.macrofit.rutinas.model

import com.google.gson.annotations.SerializedName

data class RutinaUsuarioHistorialDTO(
    val idRutinaUsuario: Int,
    val idRutina: Int,
    @SerializedName("nombre_rutina")
    val nombreRutina: String?,
    val fechaInicio: String,
    val fechaFin: String?,
    val activo: Boolean
)